packages/dds/tree/src/test/schemaFactoryAlpha.ts

```diff
@@ -0,0 +1,90 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import type { ScopedSchemaName, InsertableObjectFromSchemaRecord } from "../internalTypes.js";
+import {
+	SchemaFactory,
+	type SchemaFactoryObjectOptions,
+	type TreeNodeSchemaClass,
+	type NodeKind,
+	type Unenforced,
+	type ImplicitFieldSchema,
+} from "../simple-tree/index.js";
+// eslint-disable-next-line import/no-internal-modules
+import { defaultSchemaFactoryObjectOptions } from "../simple-tree/api/schemaFactory.js";
+// eslint-disable-next-line import/no-internal-modules
+import { type TreeObjectNode, objectSchema } from "../simple-tree/objectNode.js";
+import type { RestrictiveStringRecord } from "../util/index.js";
+
+/**
+ * Copy of {@link SchemaFactory} with additional alpha APIs.
+ *
+ * @privateRemarks
+ * Not currently exported to the public API surface as doing so produces errors in API-extractor.
```

Comment:
There are some other features that have been thrown around lately that would be implemented in a SchemaFactoryAlpha, so this will need to get lifted/shared eventually.

Reply:
Yeah. I don't remember exactly what goes wrong, but agreed we'll need to sort this out. I'm planning on lifting this explicitly directly after this PR, so we'll revisit this at that point in a PR that is more scoped :)